pkg/flash/flash_linux.go

```diff
@@ -149,10 +151,19 @@ func (f *Flash) writeAt(p []byte, off int64) (int, error) {
 		{Tx: op.PRDRES.Bytes(), CSChange: true},
 		{Tx: op.WriteEnable.Bytes(), CSChange: true},
 		// Send the address.
-		{Tx: append(op.PageProgram.Bytes(), f.prepareAddress(off)...)},
-		// Send the data.
-		{Tx: p, CSChange: true},
-		{Tx: op.WriteDisable.Bytes(), CSChange: true},
+		{Tx: append(append(op.PageProgram.Bytes(), f.prepareAddress(off)...), p...)},
+	}); err != nil {
+		return 0, err
+	}
+	time.Sleep(10 * time.Microsecond)
```

Comment:
The `Transfer` struct has a `DelayUSecs` option. `DelayUSecs is how long to delay before the next transfer (optional).`. Then you won't be affected by userspace scheduling.

Reply:
I think I'm going to replace this by polling the SR.

Reply:
Never mind, you are right. Delay Usecs is a good way to go.

For now, we can write the data at one byte per (say) 10 microseconds + 8 or so bytes of transfer, which is not horrendous. We can make it faster later.